source/markdown/what-we-do/2.md

```diff
@@ -0,0 +1,3 @@
+Mozilla strives to build and support an equitable and inclusive online world for everyone. To do this, we want to challenge the existing gender imbalances that exist and create an online space where we can reach gender equality at scale.
+
+We want to help shift the default to one where women have a strong, equitable role in reading, writing and participating on the web—and, ultimately, help lead the next wave of openness, innovation and opportunity online.
```

Comment:
The dash between "web" and "and" looks like an em dash in the redpen. Can we use that here as well? `&mdash;`